selene/tests/cves/test_get_cve.py

```diff
@@ -99,6 +122,29 @@ def test_complex_cve(self, mock_gmp: GmpMockFactory):
                     updateTime
                     description
                     products
+                    cvssVector
```

Comment:
Do we still have a cvssVector field?

Reply:
Yes. We have and will need the cvssVector [field](https://github.com/greenbone/hyperion/blob/master/selene/tests/cves/example-cve.xml#L16).
When querying infos without details, we won't have the `raw_data` and that field is then the only way to ask for the vector.
While talking about this I recognized that I have missed the `score` field ... BRB.